src/common/low_precision_transformations/src/reduce_sum.cpp

```diff
@@ -69,10 +69,23 @@ void ReduceSumTransformation::changeDequantizationValues(
 
         // (a1 - s) + (a2 - s) + ... + (an - s) = (a1 + a2 + ... + an) - n * s
         const auto reductionSizeConstant = ov::opset1::Constant::create(deqPrecision, Shape{}, { static_cast<float>(reductionSize) });
-        const auto result = fold<ov::opset1::Multiply>(dequantization.subtractConstant, reductionSizeConstant);
+        assert(deqPrecision == dequantization.subtract->get_input_element_type(0));
```

Comment:
```suggestion
        OPENVINO_ASSERT(deqPrecision == dequantization.subtract->get_input_element_type(0));
```

Reply:
I prefer `assert`, we doesn't need check for production here

Reply:
updated, thanks